components/applications-service/cmd/applications-publisher/main.go

```diff
@@ -94,6 +94,8 @@ func main() {
 	flag.StringVar(&event.PkgIdent.Name, "name", "redis", "The name of a package")
 	flag.StringVar(&event.PkgIdent.Version, "version", "0.1.0", "The version of a package")
 	flag.StringVar(&event.PkgIdent.Release, "release", t.Format("20060102150405"), "The release of a package")
+	flag.StringVar(&event.Site, "site", "", "The site of the server where the service is running")
+	flag.StringVar(&event.Channel, "channel", "", "The channel that the supervisor is subscribed to")
```

Comment:
maybe do the default channel to be `stable`?

Reply:
I want to be able to test a blank channel in the ui.